token-account-based/README.md

```diff
@@ -186,6 +211,143 @@ The function queries the balance of the account associated with the recipient cl
 
 Congratulations, you've transferred 100 tokens! The Org2 recipient can now transfer tokens to other registered users in the same manner.
 
+## Another scenario
```

Comment:
I would add (for JavaScript contract only) as part of the header here, and remove the sections below. This entire section is only relevant for the ERC-20 JavaScript contract.

Reply:
Thank you for your review. I added 'for JavaScript contract only' to the header and remove it from each command in the section.